database/migrations/2024_06_24_150910_create_switch_logs_table.php

```diff
@@ -17,7 +17,7 @@ public function up()
             $table->id();
             $table->unsignedBigInteger('user_id')->nullable();
             $table->timestamp('action_time')->useCurrent();
-            $table->foreign('user_id')->references('id')->on('users');
+            $table->foreign('user_id')->references('id')->on('set null');
```

Comment:
Diese Änderung macht leider das Datenbankschema kaputt. Schaue am besten einmal in die [Dokumentation von Laravel](https://laravel.com/docs/11.x/migrations#foreign-key-constraints) rein. Du solltest hier eine Option onDelete("SET NULL")` hinzufügen.